<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPhase</class>
 <widget class="QDialog" name="DlgPhase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фазировка модулей передатчика</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="textLabelPrd">
       <property name="text">
        <string>ПРД:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSelectPrd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>ВСЕ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ПРД-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ПРД-2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabelModule">
       <property name="text">
        <string>УМИ:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSelectModule">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maxVisibleItems">
        <number>16</number>
       </property>
       <item>
        <property name="text">
         <string>ВСЕ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-31</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-1...16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>УМИ-17...32</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>116</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAuto">
       <property name="text">
        <string>Авто</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushMeasure">
       <property name="text">
        <string>Измерить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushMiddle">
       <property name="text">
        <string>Среднее &gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabelPhase">
       <property name="text">
        <string>Фцел (град) =</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPhase"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushPhase">
       <property name="text">
        <string>Сфазировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>ПРД-1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tablePhase1">
         <column>
          <property name="text">
           <string>Номер</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Заводской КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Текущий КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Поправка КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Фаза УМИ</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>ПРД-2</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tablePhase2">
         <column>
          <property name="text">
           <string>Номер</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Заводской КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Текущий КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Поправка КФ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Фаза УМИ</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="pushCalculate">
       <property name="text">
        <string>Вычислить ПКФ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClear1">
       <property name="text">
        <string>Очистить ПКФ 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClearPhase1">
       <property name="text">
        <string>Очистить Ф 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushRefresh">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClear2">
       <property name="text">
        <string>Очистить ПКФ 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClearPhase2">
       <property name="text">
        <string>Очистить Ф 2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushApply">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>comboSelectPrd</tabstop>
  <tabstop>comboSelectModule</tabstop>
  <tabstop>checkBoxAuto</tabstop>
  <tabstop>pushMeasure</tabstop>
  <tabstop>pushMiddle</tabstop>
  <tabstop>lineEditPhase</tabstop>
  <tabstop>pushPhase</tabstop>
  <tabstop>pushCalculate</tabstop>
  <tabstop>pushClear1</tabstop>
  <tabstop>pushRefresh</tabstop>
  <tabstop>pushClear2</tabstop>
  <tabstop>pushApply</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
